src/main/java/htsjdk/samtools/util/SortingCollection.java

```diff
@@ -66,23 +66,28 @@
     public interface Codec<T> extends Cloneable {
         /**
          * Where to write encoded output
+         *
          * @param os
```

Comment:
IMO it's just clutter to have blank `@param` entries. Either we should have some text here, or we should remove `@param`. Either way the javadoc will contain the parameter name.